inbox/api/update.py

```diff
@@ -223,7 +223,7 @@ def update_message_labels(
     # XXX: Non-optimistic updates are buggy because if we don't update the
     # state and can't supply added/removed labels we'll end up with
     # inconsistencies.
-    if optimistic or True:
+    if optimistic:
```

Comment:
This branch would always run, and now it won't, right? Is this the intention here? I would expect this PR to mostly not change behavior, that's why I'm checking explicitly that this is what is intended here.

Reply:
True dat. Thanks for catching this.

I looked at the code a little bit to understand what `optimistic` is, and it looks like it's a runtime feature flag triggered by client HTTP API header. I need to double check but it looks like we would always want optimistic behavior in other places of code as well because we don't set that header ourselves.